src/main/java/seedu/souschef/logic/parser/Parser.java

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
It's inherited by XYZCommandParser classes(e.g. AddCommandParser, DeleteCommandParser, etc.). As XYZCommand classes inherits Command class, if we want to rename it CommandParser would be better. However, because it preserves original AB4's structure as I mentioned above, I think it would still be fine to just leave it as Parser

Reply:
Made change and pushed.